llvm/docs/TestSuiteGuide.md

```diff
@@ -367,6 +367,18 @@ For the SPEC benchmarks you can switch between the `test`, `train` and
 `ref` input datasets via the `TEST_SUITE_RUN_TYPE` configuration option.
 The `train` dataset is used by default.
 
+In addition to SPEC, the multimedia frameworks ffmpeg and dav1d can also
+be hooked up as external projects in the same way. Adding them doesn't
+add the respective projects' full testsuites though. But by including
```

Comment:
One nit, one possible clarification about the "doesn't add the full test suites though". I think this may leave a reader wondering what is added instead. I think you explain that further down with:

> There is a small code correctness test that compares
the output of the compiler generated functions against handwritten assembly
versions. 

So perhaps if you move this up, you can then relate or contrast this to the "doesn't add the full test suite" in previous sentence.

Reply:
Good points, thanks. Yes, it reads much better if we start out by telling you what you get from it, rather than telling you what you don't; I rewrote it in such an order now.